src/test/java/org/broadinstitute/hellbender/tools/exome/TargetCollectionUnitTest.java

```diff
@@ -279,6 +279,7 @@ public TargetCollectionStub(final int numberOfTargets, final Random rdn) {
                 final int targetLength = (int) Math.max(1,slotLength * rdn.nextDouble());
                 final int start = Math.max(nextBasePos,1 + nextBasePos + ((slotLength - targetLength) >> 1));
                 final int stop = start + targetLength;
+                // TODO - this can create intervals with stop > contigLength
```

Comment:
I haven't look at the logic of this tests, but it looks like it is generating intervals larger than the contig length - @vruano, can you help me to fix this now that the `TargetsToolsTestUtils` is factor out in favor of `IntervalUtils` methods?